src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useEffect } from 'react';
+import classNames from 'classnames';
+import { ErrorType } from '../../types/Error';
+
+type Props = {
+  error: ErrorType;
+  setError: React.Dispatch<React.SetStateAction<ErrorType>>
+};
+
+export const ErrorMessage: FC<Props> = ({ error, setError }) => {
+  const handleErrorMessageHidden = () => {
+    setError(ErrorType.NONE);
+  };
+
+  useEffect(
+    () => {
+      if (error === ErrorType.NONE) {
+        return () => {};
+      }
+
+      const timerId = setTimeout(() => setError(ErrorType.NONE), 3000);
+
+      return () => {
+        clearTimeout(timerId);
+      };
+    }, [error],
+  );
+
+  return (
+    <div
+      className={
+        classNames(
+          'notification is-danger is-light has-text-weight-normal', {
+            hidden: !error,
```

Comment:
Using '!error' might not work as expected because ErrorType.NONE is an enum value, not a falsy value. Consider using 'error !== ErrorType.NONE' instead.